src/OpenTelemetry/CHANGELOG.md

```diff
@@ -12,6 +12,8 @@
   `OTEL_DOTNET_EXPERIMENTAL_METRICS_EMIT_OVERFLOW_ATTRIBUTE` to `true` before
   setting up the `MeterProvider`.
   ([#4737](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4737))
+* Remove Jaeger Exporter from the core SDK. Jaeger supports OTLP protocol natively.
```

Comment:
```suggestion
* Remove Jaeger Exporter from the core components. Jaeger supports OTLP protocol natively.
```

I think that this message should go https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Exporter.Jaeger/CHANGELOG.md or better you can put there comment similar to https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Instrumentation.MassTransit/CHANGELOG.md and https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Instrumentation.MassTransit/README.md while all code.

Reply:
It cannot be in the jaeger readme, as that is the thing we'll be deleting 🤣 (in subsequent PRs)
The nuget readme of jaeger can document its being discontinued.

Reply:
Removed the line.

How do we change the nuget readme? Should we have an issue to keep track of the update?